docs/index.md

```diff
@@ -5,7 +5,7 @@ The E3SM documentation is organized into sections for each component model and a
 ## Component Models
 
 - [EAM](./EAM/index.md)
-- [EAMxx](./EAMxx/index.md)
+- [EAMxx](https://E3SM-project.github.io/scream)
```

Comment:
It should be removed entirely.

Reply:
Or even better make this just plain text.
`- EAMxx - not yet supported.`

If we link to EAMxx now from this page, we're implying that is somewhat supported.  We don't want that.